contexts/arbo-context.tsx

```diff
@@ -157,27 +156,65 @@ export const arboReducer = (
   }
 };
 
-const adjustMapPositionIfCountryFilterHasChanged = (
-  action: ArboAction,
-  map: MapRef
+interface GetAllBoundingBoxesFromExistingFiltersInput {
+  selectedFilters: Record<string, string[] | undefined>;
+}
+
+const getAllBoundingBoxesFromExistingFilters = (
```

Comment:
This part might be kind of hard to follow but basically the old code was pretty hacky and made an assumption that `country` was the only filter that could move the map. I need to look at the added filter as well as any existing filters now because there are two filters that determine the bounding box the map uses now. This isn't perfect looking either but yeah.

Q: Sean isn't this silly? I mean if you pick region "Eastern Europe" and "India" you'll filter out literally everything. Who cares if the map zooms in on the correct bounding box?

A: Yes you're right! This I left a discussion point on #116. I'd like to make a separate ticket to make any filters on geographical regions additive with one another so that "Eastern Europe" + "India" doesn't mean get me any "Indian  studies in Eastern Europe" and instead means "get me any studies from India or Eastern Europe".